exe/main.cc

```diff
@@ -124,9 +124,10 @@ int main(int argc, char *argv[]) {
     }
     if (!err_msg.empty()) {
         tryLogError(err_msg);
-        boost::nowide::cout << err_msg
-                            << "\nCannot start pxp-agent"
-                            << std::endl;
+        boost::nowide::cout
+            << err_msg << "\n"
```

Comment:
If logging configuration fails, there's a decent chance translation will also fail. I'm not sure it will do so gracefully.